### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Shevtsooov**
[DEMO LINK](https://Shevtsooov.github.io/react_movies-list-add-form/)

### CODE REVIEW

**Review by silchyn — CHANGES_REQUESTED** [review]
Check your DEMO link

**Comment by Shevtsooov** [comment]
@silchyn Hi, there was a cache issue, so I had to delete the `node_modules/.cache` folder an then run` :- npm run deploy` once again.

It is working now!

File: src/components/NewMovie/NewMovie.tsx [hunk 1]
```diff
@@ -1,45 +1,99 @@
...
+
+const isUrlValid = (url: string) => (pattern.exec(url));
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
```
Comment (And678) [inline src/components/NewMovie/NewMovie.tsx:18]: You can avoid using so many `useState` by combining these calls into a single formState:

```suggestion
  const [formData, setFormData] = useState(initialFormData);
```

**Review by And678 — CHANGES_REQUESTED** [review]
Nice work adding url validation, but no error messages are showing there: 
image

**Comment by Shevtsooov** [comment]
@And678 Hello, I honestly not sure in the fix I commited and it is not even clear to me 😞 .
Please check the code and let me know where the issue is.

File: src/components/NewMovie/NewMovie.tsx [hunk 2]
```diff
@@ -1,45 +1,129 @@
...
+  imgUrl: '',
+  imdbUrl: '',
+  imdbId: '',
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+
+  const [newMovieDetails, setNewMovieDetails] = useState(initialFormState);
+  // const [isUrlInvalid, setIsUrlInvalid] = useState(false);
```
Comment (dinaminahi) [inline src/components/NewMovie/NewMovie.tsx:23]:
```suggestion
```
😑

**Review by dinaminahi — APPROVED** [review]
Well done 💪

**Pull request closed without merging**